truffle/contract.go

```diff
@@ -134,6 +136,20 @@ func GetTruffleContracts(buildDir string, networkIDs []string, objects ...*model
 			}
 
 			absPath := node.AbsolutePath
+			if absPath[0] == '@' {
+				// local path
+				path, err := os.Getwd()
```

Comment:
Rename this to `localPath` so the `path` package can be imported without the `path2` alias.